common/utils/util.js

```diff
@@ -153,18 +150,35 @@ const processReplacements = (input, replacementDetails) => {
   return JSON.parse(newInput)
 }
 
+const getOrdinalIndicator = number => {
+  const englishOrdinalRules = new Intl.PluralRules('en', { type: 'ordinal' })
+  const suffixes = {
+    one: 'st',
+    two: 'nd',
+    few: 'rd',
+    other: 'th',
+  }
+  const ordinalCategory = englishOrdinalRules.select(number)
+  return suffixes[ordinalCategory]
+}
+
 const formatDateWith = pattern => isoString => {
-  const parsedDate = parseISO(isoString)
-  return isValid(parsedDate) ? format(parsedDate, pattern) : null
+  const parsedDate = DateTime.fromISO(isoString, { zone: 'utc' })
+    .setLocale('en-GB')
+    .setZone('Europe/London')
+  // This is a current workaround due to Luxon not supporting ordinal indicators
+  const updatedPattern = pattern.replace('d ', `d'${getOrdinalIndicator(parsedDate.day)}' `)
```

Comment:
Frustratingly `luxon` does not support ordinal indicators when formatting dates.. I've added this utility function to add this functionality. Hopefully this functionality gets added later down the line as the benefits of using this library currently outweigh this shortcoming